### ISSUE TITLE
[MU4 Issue] Accessibility: selection lost when copying lines

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#15998]
**Describe the bug**
When using copy & paste on a line, the selection is lost, and cannot be regained without clicking

**To Reproduce**
1. Add a pedal or other line to a score
2. Select it
3. Copy
4. Click a note somewhere else
5. Paste
6. See error: the line is copied, but nothing is selected

**Expected behavior**
I would expect either the newly selected line to be selected, or the original note

**Screenshots**
![line-copy](https://user-images.githubusercontent.com/1799936/213872664-de7c04b9-36c7-47ef-9c17-dd20b823d905.gif)

**Platform information**
 - OS: Linux

**Additional context**
This line copy/paste functionality is new and has the potential to be very useful.  However, losing the selection is not good; it messes up the possibility of then easily creating a sequence of these lines.  If you check the status bar, it seems MuseScore *thinks* the pedal is selected, but it isn't highlyed.  And press Alt+Right to move to the next element  throws you all the way to the end of the score.  However, if I press Esc to clear the selection, *then* Alt+Right functions normally.

**Comment by MarcSabatella** [comment]
Came up again on the MENVI mailing list for blind musicians as an accessibility issue.

**Comment by XiaoMigros** [comment]
Can no longer reproduce this in 4.4

**Comment by DmitryArefiev** [comment]
> Can no longer reproduce this in 4.4

Yes, confirm on my side (**4.4.0** and **4.4.3**)

https://github.com/user-attachments/assets/dfa6c0d5-2b64-4c2b-97d2-7902d116f6ac
